net/wgpeerselector/files/usr/bin/wgpeerselector

```diff
@@ -389,7 +389,7 @@ function WGPeerSelector:main()
 			if not connected_peer:has_recent_handshake() then
 				connected_peer:uninstall_from_kernel()
 				connected_peer = nil
-				log(syslog.LOG_INFO, 'Connection to '..peer.name..' lost.')
+				log(syslog.LOG_INFO, 'Connection to '..connected_peer.name..' lost.')
```

Comment:
Why are you using a variable which is set to nil here?

Reply:
At least, it is declared ;)